packages/orchestrator-ui-components/src/components/WfoBadges/WfoHeaderBadge/WfoHeaderBadge.tsx

```diff
@@ -18,7 +18,6 @@ export const WfoHeaderBadge: FC<HeaderBadgeProps> = ({
         ? {
               height: 24,
               display: 'flex',
-              paddingLeft: 0,
```

Comment:
This padding needs to stay here, it makes the buttons look different
![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/ac5ae5d6-186f-49cf-92c0-6ece48e8c119)

![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/a97fc869-27ca-40cc-bf0d-917356d5e87e)